### PR TITLE
AWS Lambda Tool implementation

### PR DESCRIPTION
**Pull request by jasondotparse**
### Summary
This tool implements the same AWS Lambda functionality which I previously added to the JS package:https://js.langchain.com/docs/modules/agents/tools/lambda_agent

### CODE REVIEW

File: langchain/utilities/awslambda.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    Docs for using:
+
+    1. pip install boto3
+    2. Create a lambda function using the AWS Console or CLI
+    3. Run `aws configure` and enter your AWS credentials
+
+    """
+
+    wolfram_client: Any  #: :meta private:
```
Comment (hwchase17) [inline langchain/utilities/awslambda.py:21]: naming?

File: langchain/utilities/awslambda.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        client = boto3.client('lambda')
+
+        values["lambda_client"] = client
+
+        return values
+
+    def run(self, query: str) -> str:
+        """Invoke Lambda function and parse result."""
+        res = self.lambda_client.invoke(
+            FunctionName="testFunction1",
```
Comment (hwchase17) [inline langchain/utilities/awslambda.py:51]: do we need to paramertrize these?

Comment (jasondotparse) [reply langchain/utilities/awslambda.py]: Yup, we definitely do.

**Review by hwchase17 — APPROVED** [review]
this looks pretty reasonable to me - i can fix lint if you think its good to land?

**Comment by jasondotparse** [comment]
> this looks pretty reasonable to me - i can fix lint if you think its good to land?

Sure, by all means!

**Comment by jasondotparse** [comment]
> > this looks pretty reasonable to me - i can fix lint if you think its good to land?
> 
> Sure, by all means!

Never mind... figured it out

**Review by hwchase17 — COMMENTED** [review]
will let @vowelparrot decide when to merge in! (we're doing a bit of a tool refactor)

**Pull request merged**